perses/app/relative_point_mutation_setup.py

```diff
@@ -89,29 +73,28 @@ class PointMutationExecutor(object):
                                                                                   splitting="V R R R O R R R V",
                                                                                   constraint_tolerance=1e-06),
                                                                                   hybrid_factory=htf, online_analysis_interval=10)
-            hss.setup(n_states=n_states, temperature=300*unit.kelvin, storage_file = reporter, lambda_protocol = lambda_protocol, endstates=False)
+            hss.setup(n_states=n_states, temperature=300*unit.kelvin, storage_file=reporter, lambda_protocol=lambda_protocol, endstates=False)
             hss.extend(n_cycles)
 
     """
     def __init__(self,
-                 receptor_filename,
-                 ligand_filename,
+                 protein_filename,
                  mutation_chain_id,
                  mutation_residue_id,
-                 proposed_residue, phase = 'complex',
-                 conduct_endstate_validation = False,
-                 ligand_index = 0,
-                 forcefield_files = ['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml'],
-                 barostat = openmm.MonteCarloBarostat(1.0 * unit.atmosphere, temperature, 50),
-                 forcefield_kwargs = {'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.PME, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus},
-                 small_molecule_forcefields = 'gaff-2.11',
+                 proposed_residue,
+                 phase='complex',
+                 conduct_endstate_validation=False,
+                 ligand_filename=None,
+                 ligand_index=0,
+                 forcefield_files=['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml'],
+                 barostat=openmm.MonteCarloBarostat(1.0 * unit.atmosphere, temperature, 50),
+                 forcefield_kwargs={'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.PME, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus},
```

Comment:
nonbondedMethod should be passed in as a `periodic_forcefield_kwargs`

like is done here:
https://github.com/choderalab/perses/blob/8a560c332f21bacd268d6fa195ffc7551888aa6d/perses/app/relative_setup.py#L267

Reply:
Done!

Reply:
> 2020-05-29T19:47:31.0221215Z ERROR: Propose a mutation to remain at WT in insulin
2020-05-29T19:47:31.0222219Z ----------------------------------------------------------------------
2020-05-29T19:47:31.0223057Z Traceback (most recent call last):
2020-05-29T19:47:31.0224104Z   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
2020-05-29T19:47:31.0224965Z     self.test(*self.arg)
2020-05-29T19:47:31.0225824Z   File "/home/runner/work/perses/perses/perses/tests/test_topology_proposal.py", line 401, in test_propose_self
2020-05-29T19:47:31.0226708Z     system_generator = create_simple_protein_system_generator()
2020-05-29T19:47:31.0227633Z   File "/home/runner/work/perses/perses/perses/tests/test_topology_proposal.py", line 153, in create_simple_protein_system_generator
2020-05-29T19:47:31.0228712Z     small_molecule_forcefield = 'gaff-2.11', molecules=None, cache=None)
2020-05-29T19:47:31.0229876Z   File "/usr/share/miniconda/envs/test/lib/python3.6/site-packages/openmmforcefields/generators/system_generators.py", line 183, in __init__
2020-05-29T19:47:31.0230986Z     or nonperiodic_forcefield_kwargs (if it should be applied to non-periodic systems)""")
2020-05-29T19:47:31.0231872Z ValueError: nonbondedMethod cannot be specified in forcefield_kwargs;
2020-05-29T19:47:31.0232793Z                  must be specified in either periodic_forcefield_kwargs (if it should be applied to periodic systems)
2020-05-29T19:47:31.0233933Z                  or nonperiodic_forcefield_kwargs (if it should be applied to non-periodic systems)

Reply:
is this new?